server/services/health.js

```diff
@@ -10,9 +10,9 @@ function createHealthService({ client, config, logger }) {
   const { UP, DOWN, PARTIALLY_DEGRADED } = statuses;
 
   function allOk(...args) {
-    const fn = s => s === UP;
-    const all = args.every(fn);
-    const some = args.some(fn);
+    const statusIsOkay = s => s === UP;
+    const all = args.every(statusIsOkay);
+    const some = args.some(statusIsOkay);
```

Comment:
❤️ for variable names that mean something

Reply:
😄